packages/portfolio-deploy/test/portfolio.test.ts

```diff
@@ -331,65 +331,48 @@ test.skip('CCTP settlement works', async t => {
   await controllerWallet.invokeEntry({
     id: inviteId,
     targetName: 'ymax0.creatorFacet',
-    method: 'deliverResolverInvitation',
+    method: 'deliverResolverServiceInvitation',
     args: [beneficiary, postalService],
   });
 
-  const currentWalletRecord = await wallet.getCurrentWalletRecord();
-
-  t.log('Using resolver invitation to get invitationMaker');
+  t.log('Redeeming resolver service invitation');
   await wallet.executeOffer({
-    id: 'settle-cctp',
+    id: 'resolver-1',
     invitationSpec: {
       source: 'purse',
-      instance: currentWalletRecord.purses[0].balance.value[0].instance,
-      description: 'resolver',
+      instance: agoricNamesRemotes.instance.ymax0,
+      description: 'resolver service',
     },
-    proposal: { give: {}, want: {} },
+    proposal: {},
+    saveResult: { name: 'resolver' },
   });
 
   await eventLoopIteration();
 
+  const settleId = Date.now().toString();
   t.log('Executing CCTP settlement offer');
-  await wallet.executeOffer({
```

Comment:
again, I'd prefer to keep the old test in place and only add a new one. would that be a huge hassle?

Reply:
These tests are skipped and broken (the offer args don't match shape). Keeping the old ones don't do anything.

Reply:
I see.